G/Glib/build_tarballs.jl

```diff
@@ -95,14 +95,14 @@ dependencies = [
     # Host gettext needed for "msgfmt"
     HostBuildDependency("Gettext_jll"),
     Dependency("Libiconv_jll"),
-    Dependency("Libffi_jll", v"3.2.2"; compat="~3.2.2"),
+    Dependency("Libffi_jll", v"3.2.2"; compat="~3.2.2"), # would need 3.4.6 for aarch64-*-freebsd
     # Gettext is only needed on macOS, as far as I could see
     Dependency("Gettext_jll", v"0.21.0"; compat="=0.21.0"),
-    Dependency("PCRE2_jll"; compat="10.35"),
+    Dependency("PCRE2_jll"; compat="10.43"),
```

Comment:
PCRE2 is a Julia dependency, this would make the package uninstallable

Reply:
Yes, I realized that later.